content/tools/ndmg.yaml

```diff
@@ -6,7 +6,7 @@ name: ndmg
 github: "neurodata/ndmg"
 docs: "https://ndmg.neurodata.io/"
 installation: "https://ndmg.neurodata.io/tutorials/install.html#installation-guide"
-examples: "https://ndmg.neurodata.io/"
+examples: "https://github.com/neurodata/ndmg/tree/staging/tutorials"
```

Comment:
(Eventually) should put the examples into the documentation site, no?

Reply:
yep, agreed